scripts/e2e/config/knative/overlay.yaml

```diff
@@ -121,27 +121,3 @@ metadata:
 spec:
   minReplicas: 1
   maxReplicas: 1
-
----
-# Copyright 2019 The Knative Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     https://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-apiVersion: v1
```

Comment:
Why is this configmap now removed?

Reply:
Originally, we needed this config-deployment configmap to add `dev.localhost:1234` in `registries-skipping-tag-resolving` so Merlin-related docker images can be resolved. However, I've tested to remove it and it still works.